### DISCUSSION TITLE
How to temporarily disable multi-level tags? 怎样暂时停止多级标签?

### DISCUSSION THREAD

**Discussion by Zhou-Shangjie** [usememos/memos#1818] [Q&A] [answered]
Hello! I want to create a tag which contains a slash (See the pic below. The tag I want to create is just "AdS/CFT"), but memos automatically convert it into a multi-level tag. What should I do to temporarily disable multi-level tags? 

我想创建一个包含斜杠的标签（如图所示，我想创建的是“AdS/CFT”），但是memos自动转换成了一个多级标签。怎样才能暂时停止多级标签。感谢帮助！
![Imgur](https://i.imgur.com/0Uyp3Cd.png)

**Comment by Zeng1998** [comment, marked as answer]
not support.
